Catalog/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Common.Models.Comments;
+using Catalog.Services.Comments.Interfaces;
+
+namespace Catalog.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CommentsController : ControllerBase
+    {
+        private readonly ICommentsRepository _commentServices;
+        public CommentsController()
```

Comment:
A controller is the only entry point for any kind of interaction with API. With having it completely blank, we can't use Comments logic even though it is practically ready for use.
- [ ] Make sure to cover your Service methods with respective methods in the CommentsController.